src/explorer/DeploymentSlotTreeItem.ts

```diff
@@ -30,17 +25,4 @@ export class DeploymentSlotTreeItem extends SiteTreeItem {
     public get iconPath(): IThemedIconPath {
         return getThemedIconPath('DeploymentSlot_color');
     }
-    public promptToDeploy(context: IActionContext): void {
-        const createdNewSlotMsg: string = `Created new slot "${this.root.client.fullName}": https://${this.root.client.defaultHostName}`;

```

Comment:
> I know that'll make Eric happy

I think we should keep this notification 😬😬😬 It's a long running operation and I like having notifications when those complete. (just _one_ notification though haha)

I have less strong feelings about the "Deploy" button. I don't like it in the "Create Web App" scenario because there's a clear alternative to create/deploy at the same time (the big blue up arrow). There is no up arrow to create/deploy a slot at the same time. But I also agree that I don't think many people will do that so idk